app/controllers/conversations_controller.rb

```diff
@@ -63,16 +66,18 @@ def create
 
   def show
     respond_to do |format|
-      format.html do
-        redirect_to conversations_path(:conversation_id => params[:id])
-        return
+      unless params[:raw]
```

Comment:
This feels like it misuses this route/action as two, perhaps better add a new route.